### ISSUE TITLE
In Moodle, the cookie can have a prefix. If one is set, the session sent to SEB server is not correct.

### ISSUE THREAD

**Issue by lucaboesch** [SafeExamBrowser/seb-mac#396]
This issue duplicates the one for SEB Windows, https://github.com/SafeExamBrowser/seb-win-refactoring/issues/848.
The mechanism getting the user ID for Shibolleth authenticated Moodle users should fetch /mod/quiz/accessrule/sebserver/classes/external/user.php of if that isn't around, /theme/boost_ethz/sebuser.php and get those addresses providing a Session cookie. The session cookie can have a variable name and isn't always "MoodleCookie", but can vary.
Refer to https://github.com/SafeExamBrowser/seb-win-refactoring/blob/master/SafeExamBrowser.Browser/Handlers/ResourceHandler.cs to see how it is dealt with in SEB Windows.
Thanks.

**Comment by lucaboesch** [comment]
Dear @danschlet 

It does not seem to me that the issue is resolved.
In my place the authentication does not fully pass and the user name is not propagated (it sticks to the machine's name)

![06-01-2025_14-38-09](https://github.com/user-attachments/assets/3483aaa9-6eaf-44dc-8645-cb5fed59ce44)

**Comment by danschlet** [comment]
Yeah, that's the problem when people use nonstandard setups. Can you provide me access to a test system/account?
